### PR TITLE
Adding files for further development of menu and features

### PR DESCRIPTION
**Pull request by davidberard2**
I added addition fragment and favorite fragment. Daniel will therefore be able to implement addition of files and Alex you'll be able to develop menu.

### CODE REVIEW

**Review by Alex-Hayeur — APPROVED** [review]
Appropriate names, thanks for making the shell files! They'll be a big help to prevent bottlenecking.

File: app/src/main/java/Fragments/AdditionFragment.java [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+import android.os.Bundle;
+import android.support.v4.app.Fragment;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+
+import com.projectfirebase.soen341.root.R;
+
+public class AdditionFragment extends Fragment {
+
```
Comment (kkhh001) [inline app/src/main/java/Fragments/AdditionFragment.java:13]: Good initiative to avoid possible discrepancy during implementation 👍

**Review by kkhh001 — APPROVED** [review]
Good blueprint design, will avoid bottleneck as @Alex-Hayeur mentioned

**Review by LHuss — APPROVED** [review]
Good job on taking initiative to avoid possible problems!

**Pull request merged**